jupyter/functional-test.ipynb

```diff
@@ -0,0 +1,154 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Jupyter notebook example to play with functional tests on k8s backend\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"BUILD_USER_EMAIL\"] = \"sct.tester@scylladb.com\"\n",
+    "os.environ[\"BUILD_ID\"] = \"123\"\n",
+    "os.environ[\"SCT_CLUSTER_BACKEND\"] = \"k8s-local-kind\"\n",
+    "os.environ[\"SCT_CONFIG_FILES\"] = \"test-cases/scylla-operator/functional.yaml\"\n",
+    "os.environ[\"SCT_SCYLLA_VERSION\"] = \"2022.2.0\"\n",
+    "os.environ[\"SCT_K8S_USE_CHAOS_MESH\"] = \"True\"\n",
+    "os.environ[\"SCT_REGION_NAME\"] = \"eu-west-1\"\n",
+    "os.environ[\"SCT_USE_MGMT\"] = \"True\"\n",
+    "os.environ[\"SCT_N_DB_NODES\"] = \"3\"\n",
+    "os.environ[\"SCT_N_LOADERS\"] = \"1\"\n",
+    "os.environ[\"SCT_N_MONITORS_NODES\"] = \"0\"\n",
+    "assert os.environ[\"BUILD_USER_EMAIL\"] != \"sct.tester@scylladb.com\", \"please use your own email so resources are tracked properly\"\n",
+    "\n",
+    "# logging configuration, for jupyter only (sct logs are intact)\n",
+    "import logging\n",
+    "\n",
+    "LOGGER = logging.getLogger(__name__)\n",
+    "LOGGER.setLevel(logging.DEBUG)\n",
+    "consoleHandler = logging.StreamHandler()\n",
+    "consoleHandler.setLevel(logging.DEBUG)\n",
+    "formatter = logging.Formatter('%(asctime)s - %(name)s - %(levelname)s - %(message)s')\n",
+    "consoleHandler.setFormatter(formatter)\n",
+    "LOGGER.addHandler(consoleHandler)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "from functional_tests.scylla_operator.libs.auxiliary import ScyllaOperatorFunctionalClusterTester\n",
+    "from sdcm import sct_abs_path\n",
+    "\n",
+    "os.chdir(sct_abs_path())\n",
```

Comment:
sct_abs_path has required parameter. Will it work without parameter?

Reply:
yes, fixed.